package.json

```diff
@@ -73,5 +75,9 @@
     "truffle": "^4.1.3",
     "web3-utils": "^1.0.0-beta.30"
   },
-  "dependencies": {}
+  "dependencies": {
+    "solidity-sha3": "^0.4.1",
+    "truffle-contract": "^3.0.2",
+    "web3": "0.20.6"
```

Comment:
JS client is using `ethers-js` so perhaps be consistent with that? https://github.com/ethers-io/ethers.js/blob/8a842af9993185c9d098698464c01192cadba146/providers/json-rpc-provider.js#L59

Reply:
An adapter for web3 is also in the making. Either way I'm wondering why web3 1.0 isn't being used here.